base/mgr/job.go

```diff
@@ -65,6 +67,11 @@ func newJob(command *pm.Command, factory ProcessFactory, hooks ...pm.RunnerHook)
 
 		registerPID: registerPID,
 		waitPID:     waitPID,
+		backoff: BackOff{
+			Delay:    2 * time.Second,
+			Multiply: 1.5,
+			Max:      5 * time.Minute,
```

Comment:
So this is going to be set by default for all processes ?

Reply:
Only protected processes (this include redis, and libvirtd, etc ...)